neqo-crypto/src/agent.rs

```diff
@@ -1040,7 +1040,7 @@ impl Client {
         let mut client = Self {
             agent,
             server_name,
-            resumption: Box::pin(Vec::new()),
+            resumption: Box::pin(Vec::with_capacity(256)), // Typical TLS resumption token size
```

Comment:
This number is very wrong.  And trivially measurable.  We should be able to get traces with a better number.  Any of our resumption tests, with logging on, will spit out a value that we can round up by a little bit.

Reply:
Tokens are 840 or 856 bytes in tests.

Reply:
I'd hedge and say 900.